### ISSUE TITLE
can't update to 5.0.19 fromt 5.0.18

### ISSUE THREAD

**Issue by kapkirk** [home-assistant/addons#3653]
### Describe the issue you are experiencing

Home Assistant OS report:
"Can't install homeassistant/amd64-addon-letsencrypt:5.0.19: 404 Client Error for http+docker://localhost/v1.45/images/create?tag=5.0.19&fromImage=homeassistant%2Famd64-addon-letsencrypt&platform=linux%2Famd64: Not Found ("manifest for homeassistant/amd64-addon-letsencrypt:5.0.19 not found: manifest unknown: manifest unknown")"

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

CEC Scanner

### What is the version of the add-on?

5.018

### Steps to reproduce the issue

1.
2.
3.
...

## System Information

version | core-2024.6.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.2
os_name | Linux
os_version | 6.6.33-haos
arch | x86_64
timezone | Europe/Rome
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.34.0
Stage | running
Available Repositories | 1474
Downloaded Repositories | 76
HACS Data | ok

</details>

<details><summary>AccuWeather</summary>

can_reach_server | ok
-- | --
remaining_requests | 24

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 1 gennaio 2018 01:00
relayer_connected | false
relayer_region | null
remote_enabled | false
remote_connected | false
alexa_enabled | false
google_enabled | false
remote_server | null
certificate_status | null
instance_id | 0bbab73a9d3d473e9599b24f668a3e67
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 12.4
-- | --
update_channel | stable
supervisor_version | supervisor-2024.06.0
agent_version | 1.6.0
docker_version | 26.1.4
disk_total | 234.0 GB
disk_used | 23.3 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | 
board | generic-x86-64
supervisor_api | ok
version_api | ok
installed_addons | eWeLink Smart Home (1.4.3), Advanced SSH & Web Terminal (18.0.0), Dynu DNS (1.0.0), Home Assistant Google Drive Backup (0.112.1), ESPHome (2024.3.1), DSS VoIP Notifier (4.0.0), Samba share (12.3.1), MariaDB (2.7.1), AppDaemon (0.16.6), Network UPS Tools (0.13.0), File editor (5.8.0), Whisper (2.1.0), Piper (1.5.0), Zircon3D Proxy (0.3.2)

</details>

<details><summary>Dashboards</summary>

dashboards | 3
-- | --
resources | 91
views | 39
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 11 giugno 2024 15:16
-- | --
current_recorder_run | 21 giugno 2024 11:27
estimated_db_size | 1201.59 MiB
database_engine | sqlite
database_version | 3.44.2

</details>

### Anything in the Supervisor logs that might be useful for us?

```txt
2024-06-21 11:26:33.495 ERROR (MainThread) [supervisor.docker.interface] Can't install homeassistant/amd64-addon-letsencrypt:5.0.19: 404 Client Error for http+docker://localhost/v1.45/images/create?tag=5.0.19&fromImage=homeassistant%2Famd64-addon-letsencrypt&platform=linux%2Famd64: Not Found ("manifest for homeassistant/amd64-addon-letsencrypt:5.0.19 not found: manifest unknown: manifest unknown")
```

### Anything in the add-on logs that might be useful for us?

```txt
Registratore: homeassistant.components.hassio
Fonte: components/hassio/websocket_api.py:135
Integrazione: Home Assistant Supervisor (documentazione, problemi)
Prima occorrenza: 11:28:29 (3 occorrenze)
Ultima registrazione: 11:32:40

Failed to to call /store/addons/core_letsencrypt/update - Can't install homeassistant/amd64-addon-letsencrypt:5.0.19: 404 Client Error for http+docker://localhost/v1.45/images/create?tag=5.0.19&fromImage=homeassistant%2Famd64-addon-letsencrypt&platform=linux%2Famd64: Not Found ("manifest for homeassistant/amd64-addon-letsencrypt:5.0.19 not found: manifest unknown: manifest unknown")
Failed to to call /addons/core_letsencrypt/install - Can't install homeassistant/amd64-addon-letsencrypt:5.0.19: 404 Client Error for http+docker://localhost/v1.45/images/create?tag=5.0.19&fromImage=homeassistant%2Famd64-addon-letsencrypt&platform=linux%2Famd64: Not Found ("manifest for homeassistant/amd64-addon-letsencrypt:5.0.19 not found: manifest unknown: manifest unknown")
```

**Comment by kapkirk** [comment]
The same error also occurs after uninstalling. Now i can't install it.

**Comment by Fr4mby** [comment]
Same issue here. Can't update from 5.0.18 to 5.0.19

**Comment by VlastiBroucek** [comment]
Same here:
Can't install homeassistant/amd64-addon-letsencrypt:5.0.19: 404 Client Error for http+docker://localhost/v1.45/images/create?tag=5.0.19&fromImage=homeassistant%2Famd64-addon-letsencrypt&platform=linux%2Famd64: Not Found ("manifest for homeassistant/amd64-addon-letsencrypt:5.0.19 not found: manifest unknown: manifest unknown")

**Comment by FrankGiesecke** [comment]
I found the reason: The tag 5.0.19 is missing In Docker Hub

https://hub.docker.com/r/homeassistant/aarch64-addon-letsencrypt/tags

There is now a tag for 5.0.20 but Home Assistant currently didn't recognize the new version.

**Comment by VlastiBroucek** [comment]
Tried update again and it has now updated to 5.0.20

**Comment by Igor-Yavych** [comment]
> I found the reason: The tag 5.0.19 is missing In Docker Hub
> 
> https://hub.docker.com/r/homeassistant/aarch64-addon-letsencrypt/tags
> 
> There is now a tag for 5.0.20 but Home Assistant currently didn't recognize the new version.

It does. You just need to run a check for an update

**Comment by kapkirk** [comment]
Sono in riunione, parliamo più tardi



In data 21 giugno 2024 13:46:26 Igor Yavych ***@***.***> ha 
scritto:

**Comment by agners** [comment]
Yeah #3549 included some errors I've not noticed. Things should be fixed and back working fully with 5.0.21.
